### PR TITLE
create(Prisma.Schema): added relations....

### PR DESCRIPTION
**Pull request by Sleepingpirates**

### CODE REVIEW

**Review by rob-3 — CHANGES_REQUESTED** [review]
I think frontend decided to go for saves instead of likes, so if you could rename the relevant fields this looks good.

**Comment by Sleepingpirates** [comment]
> I think frontend decided to go for saves instead of likes, so if you could rename the relevant fields this looks good.

saves are different from likes

**Comment by rob-3** [comment]
Wait I see that you're doing saves into collections and then likes separately. I think that should be ok but I'll run it by them.

**Comment by Sleepingpirates** [comment]
> Wait I see that you're doing saves into collections and then likes separately. I think that should be ok but I'll run it by them.

I know its not MVP but I added it for the future

**Comment by rob-3** [comment]
We can always remove it in the future. When you merge run the prisma migration please!

**Review by rob-3 — DISMISSED** [review]

File: web/src/pages/api/createPost.ts [hunk 1]
```diff
@@ -49,8 +52,17 @@ export default async function createPost(req: NextApiRequest, res: NextApiRespon
...
     }
+    //response data into json
 
     const resData = await data.json();
 
-    return(res.json({'image': resData.image}));
+    const post = await prisma.post.create({
+        data: {
+            prompt: query.prompt.toString(),
+            imageURL: resData.image as string,
```
Comment (rob-3) [inline web/src/pages/api/createPost.ts:62]: I don't think this cast is necessary to pass the linting.

**Review by rob-3 — CHANGES_REQUESTED** [review]

**Comment by rob-3** [comment]
Wait, what I actually meant is that you can do:
```ts
            prompt: query.prompt.toString(),
            imageURL: resData.image,
            authorId: session.user?.id as string
```

**Review by rob-3 — APPROVED** [review]

**Pull request merged**